ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
@aaronlehmann I'm currently making 2 updates:

1.  Downloading a new TLS cert, I'm making sure that the TLS cert verifies against both the old and the new root (because it always should, otherwise the new TLS certs may only validate against some nodes and not others)

2. The RootCA is only replaced if the node is a worker.  I'm updating managers to automatically update their own RootCA from the raft store - I'm currently only doing it if the manager is not a leader, because otherwise the ca server watch and the manager watch may update the RootCA, but since they will update to the same value that's probably ok (I'm adding the check to eliminate duplicate effort).

I think it makes sense for all managers to contact the ca server even if they can sign in order to get new certificates.  It seems simpler and safer just to gate all renewals through the CA server. 

Otherwise, we would need to be very careful in revoking the signing capabilities of any manager's RootCA object when it gets demoted, and/or `RenewTLSConfigNow` will need to check to see what the current role is before calling either `rootCA.RequestAndSaveNewCertificates` or `rootCA.IssueAndSaveNewCertificates`.